include/libnuraft/log_entry.hxx

```diff
@@ -38,12 +40,21 @@ public:
     log_entry(ulong term,
               const ptr<buffer>& buff,
               log_val_type value_type = log_val_type::app_log,
-              uint64_t log_timestamp = 0)
+              uint64_t log_timestamp = 0,
+              uint32_t crc32 = 0,
+              bool compute_crc = true)
```

Comment:
Can this be `false` by default?

Reply:
This is to cope with conf logs.  There are more than 1 place where conf logs are constructed. Either we need to copy the CRC computation code (or this true flag) everywhere (and I need to make sure I found every place), or we bite the performance overhead and lean towards safety.